extensions/wikia/SpecialEditHub/SpecialEditHub.setup.php

```diff
@@ -46,7 +46,5 @@
 
 $wgGroupPermissions['*']['edithub'] = false;
 $wgGroupPermissions['staff']['edithub'] = true;
-$wgGroupPermissions['sysop']['edithub'] = true;
```

Comment:
@owend Right now edithub allows local sysops to change marketing/core hub pages. We want to have volunteer sysops to allow for general curation, but it may just be cleaner in the long run not to give them this power. If it proves later that we want to change course, we can just put it back in.